### PR TITLE
Reduce post styles and add blog header images 💪

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
Reduce post styles and make em more consistent between another. 
Includes blog header images!

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
blog-post-header-image

**Comment by RonaldDijkstra** [comment]
Blogs can have a header image but have the Defacto main mood image as default

**Comment by RonaldDijkstra** [comment]
Ready @sn3p 

Probably a bit big this PR, I'm sorry for that.

**Comment by sn3p** [comment]
> Probably a bit big this PR, I'm sorry for that.

Awesome, will check in a bit 👍

File: source/blog/nl/2018-01-17-5-moments-of-need.html.markdown [hunk 1]
```diff
@@ -5,6 +5,7 @@ tags: onderwijskunde, performance support
 author: Joep
 lang: nl
 featured: true
+mood_image: images/blog/social/5-moments-of-need-1200x630.jpg
```
Comment (sn3p) [inline source/blog/nl/2018-01-17-5-moments-of-need.html.markdown]: What is the difference between `mood_image` and `image`?

Comment (RonaldDijkstra) [reply source/blog/nl/2018-01-17-5-moments-of-need.html.markdown]: mood_image is image for post-header background 
image is for social stuff and the likes

I misused this picture for testing purposes, should've been removed

**Comment by RonaldDijkstra** [comment]
thx

**Pull request merged**